src/components/CarouselImage/index.js

```diff
@@ -128,19 +135,19 @@ class Item extends Component {
                     className="rainbow-carousel-image"
                     tabIndex={this.getTabIndex()}
                     ref={this.itemRef}>
-                    <div>
-                        <img className="rainbow-carousel-image_image" src={src} alt={assistiveText} />
+                    <div className="rainbow-carousel-image_content-image-container">
+                        <div className="rainbow-carousel-image_image" style={this.getImageSrc()} alt={assistiveText} />
```

Comment:
the alt attribute is only for img element so here we need to use the AssistiveText component in order to screen readers take the assistive text. Then:
```
<div className="rainbow-carousel-image_content-image-container">
    <div className="rainbow-carousel-image_image" style={this.getImageSrc()} />
    <AssistiveText text={assistiveText} />
</div>
```